recipes/hidapi/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class HidapiConan(ConanFile):
+    name = "hidapi"
+    description = "HIDAPI is a multi-platform library which allows an application to interface " \
+                  "with USB and Bluetooth HID-Class devices on Windows, Linux, FreeBSD, and macOS."
+    topics = ("libusb", "hid-class", "bluetooth")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libusb/hidapi"
+    license = "GPL-3-or-later", "BSD-3-Clause"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+    generators = "pkg_config"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == "Windows" and self.settings.arch == "x86_64":
+            raise ConanInvalidConfiguration("There's no support for x86_64 on Windows yet, please use x86")
```

Comment:
```suggestion
        if self.settings.os == "Windows" and self.settings.arch == "x86_64":
            raise ConanInvalidConfiguration("There's no support for x86_64 on Windows yet, please use x86")
```

I didn't check on Windows, but what's the problem with x64?

Reply:
https://github.com/libusb/hidapi/blob/6a01f3b4a8862b19a7ec768752ebcfc1a412f4b1/windows/hidapi.sln#L12-L15
x86_64 (x64) is present in the visual studio solution.

Reply:
Oh, yes, x64 is added 2 years ago, when I create this recipe initially, x64 is not existed in `sln` file yet.

I just enable it and test on an azure pipeline.